doc/pages/JenkinFeatures.md

```diff
@@ -137,7 +137,7 @@ Download any artifacts you need from your run as soon as it finishes to attach t
 - When reporting failures in issues, understand that Grinder jobs are transient (not kept for long on Jenkins).
 Sharing a link to the Grinder is only useful for a short time.
 Share the "Rerun in Grinder" link which has details of the impl/version/test target, output statistics & test artifacts in the issue; those are more concrete items when reproducing failures.
-For openjdk test failures, here are some useful triage [instructions](https://github.com/adoptium/aqa-tests/wiki/Guidance-for-Creating-OpenJDK-Test-Defects).
+For openjdk test failures, here are some useful triage [instructions](https://github.com/adoptium/aqa-tests/wiki/Guidance-for-Creating-OpenJDK-Test-Defects). Rerun failed openjdk subtest function is also available, and the rerun target name depends on the failed test name, e.g. `jdk_math` rerun subtest uses test target `jdk_custom` and failed subtest name(s), while `hotspot_compiler` uses `hotspot_custom` instead.
```

Comment:
I would add
```
For openjdk, the test target name starts with `jdk_` would be set into auto-rerun target `jdk_custom`. Otherwise, it would be set into auto-rerun target `hotspot_custom`. 

```

Reply:
Updated. Thanks.